oeps/best-practices/oep-0067-bp-tools-and-technology.rst

```diff
@@ -181,6 +181,21 @@ Frontend Technology Selection
    Documentation on how to configure Renovate automation on a repository is available
    in the `Upgrade Automation How-to`_.
 
+#. **JavaScript projects should use caret ranges for npm dependencies**
+
+   **Rationale**: Using caret ranges (e.g., ``"^1.2.3"``) in our package.json more clearly
```

Comment:
nit:

```suggestion
   **Rationale**: Using caret ranges (e.g., ``"^1.2.3"``) in our ``package.json`` more clearly
```

Reply:
Noticed at least one other instance that needed the quotes, too. Updated!